lib/sms.js

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const libpn = require('google-libphonenumber')
+const Nexmo = require('nexmo')
+const P = require('bluebird')
+const SENDER_IDS = new Map(require('./sms-sender-ids.json'))
+
+module.exports = (log, translator, templates, config) => {
+  const nexmo = new Nexmo({
+    apiKey: config.apiKey,
+    apiSecret: config.apiSecret
+  })
+  const sendSms = P.promisify(nexmo.message.sendSms, { context: nexmo.message })
+
+  // TODO: Sort out the real link
+  const LINK = 'https://moz.la/vwxyz'
```

Comment:
Hey, I recommend looking at this document: https://docs.google.com/spreadsheets/d/151_fWwEPxmPN_AbHCp-sqAkshDCLaABRrVaowxoFS10/edit#gid=0

This is our Adjust master tracker for owned channels. You should both have edit access now.

There are some rows with FxA in there. You will see the nomenclature of the URLs.

I recommend shortner too since it would be too long in SMS.